### ISSUE TITLE
Add Multiple Photo Support With New CLI

### ISSUE THREAD

**Issue by bthelin** [philipbl/Day-One-Lightroom-Plugin#26]
It looks like DayOne now has a new Command Line Interface (CLI) that supports multiple photos.  Does this mean multiple photo functionality could be added to this great plugin?

**Comment by philipbl** [comment]
Yes! I saw the update last night and will look into it this week. This plugin is in desperate need of some love.
